dataikuapi/fm/instances.py

```diff
@@ -357,7 +362,7 @@ def set_elastic_ip(self, enable, elasticip_allocation_id):
         Set a public elastic ip for this instance
 
         :param boolan enable: Enable the elastic ip allocation
-        :param str elaticip_allocation_id: AWS ElasticIP allocation ID
+        :param str elasticip_allocation_id: AWS ElasticIP allocation ID
```

Comment:
This line, just commenting here because it won't let me add one on the correct line:

On line 407, there's `` This class should not be created directly. Instead, use :meth:`FMInstance.get_info` ``, looks like the method should actually be `FMInstance.get_status`

Reply:
fixed